tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -173,8 +173,8 @@ public function theUserDisablesUserToUsingTheGraphApi(string $byUser, string $us
 	 * @throws Exception
 	 */
 	public function theUserHasDisabledUser(string $byUser, string $user): void {
-		$this->theUserDisablesUserToUsingTheGraphApi($byUser, $user);
-		$this->featureContext->thenTheHTTPStatusCodeShouldBe(200);
+		$response = $this->editUserUsingTheGraphApi($byUser, $user, null, null, null, null, false);
```

Comment:
PR states `OcisConfigContext` but the change is only in `GraphContext` :thinking: